ext/openssl/extconf.rb

```diff
@@ -184,7 +184,7 @@ def find_openssl_library
 
 Logging::message "=== Checking done. ===\n"
 
-if openssl_3
+if openssl_3 && !$mswin
```

Comment:
Is this needed?
`$warnflags` should not have this flag on mswin builds, I think.

Reply:
I can check again, but as I recall `$warnflags` was nil, so on the next line `$warnflags.sub!` throws?

Reply:
Indeed, it’s forgotten in win32/makefile.sub.
Then isn’t it better to check that variable itself?

Reply:
```suggestion
if openssl_3 && $warnflags
```

Reply:
I'll defer to your judgement.  Wasn't sure if it should be changed in mkmf.rb or elsewhere...

Reply:
Good suggestion.  I'll update.